fuzz/Cargo.toml

```diff
@@ -17,7 +17,11 @@ target-lexicon = "0.11"
 peepmatic-fuzzing = { path = "../cranelift/peepmatic/crates/fuzzing", optional = true }
 wasmtime = { path = "../crates/wasmtime" }
 wasmtime-fuzzing = { path = "../crates/fuzzing" }
-wasm-smith = "0.1.5"
+wasm-smith = "0.1.12"
+
+[features]
+default = []
```

Comment:
It should be ok to remove `default` here too

Reply:
Done.